tests/unit/plugins/module_utils/test_attachment.py

```diff
@@ -0,0 +1,681 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import os
+import sys
+
+import pytest
+from mock import ANY
+
+from tempfile import mkstemp
+
+from ansible_collections.servicenow.itsm.plugins.module_utils import errors, attachment
+from ansible_collections.servicenow.itsm.plugins.module_utils.client import Response
+
+pytestmark = pytest.mark.skipif(
+    sys.version_info < (2, 7), reason="requires python2.7 or higher"
+)
+
+FILE_DICT = {
+    "path": "some/path/file_name.txt",
+    "name": "attachment_name.txt",
+    "type": "text/markdown",
+    "hash": "hash",
+}
+
+FILE_DICT_DICT = {
+    "attachment_name.txt": {
+        "path": "some/path/file_name.txt",
+        "type": "text/markdown",
+        "hash": "hash",
+    },
+    "another_file_name.txt": {
+        "path": "some/path/another_file_name.txt",
+        "type": "text/plain",
+        "hash": "hash",
+    },
+}
```

Comment:
Please inline those definitions. I know copy-pasting them around is a bad practice when it comes to regular code, but tests are exceptions because I need to be able to compare input and output in order to be able to determine what went wrong.